### ISSUE TITLE
OpenCode credentials not detected although CLI auth works; task stalls at `OpenCode step started.`

### ISSUE THREAD

**Issue by 7panni** [freestylefly/wesight#78]
## Problem
WeSight reports **“本机凭据未检测到本机登录或密钥”** for OpenCode, even though OpenCode CLI itself detects credentials and can run normally.

In WeSight, an OpenCode task only shows:

```text
OpenCode step started.
```

and does not proceed.

## Environment

- macOS
- OpenCode binary: `/opt/homebrew/bin/opencode`
- OpenCode config directory: `~/.config/opencode`
- Config files present: `opencode.json`, `opencode.jsonc`
- Credentials file: `~/.local/share/opencode/auth.json`

## Verification

`/opt/homebrew/bin/opencode auth list` returns:

```text
Credentials ~/.local/share/opencode/auth.json

DeepSeek api
Google api

Environment

DeepSeek DEEPSEEK_API_KEY
Google GEMINI_API_KEY
```

And this works successfully:

```bash
/opt/homebrew/bin/opencode run "只回复OK"
```

So OpenCode itself can detect and use the credentials correctly.

`~/.config/opencode/opencode.json` currently contains only:

```json
{
  "$schema": "https://opencode.ai/config.json"
}
```

## Expected behavior

WeSight should detect the credentials that OpenCode already recognizes, and OpenCode tasks should continue past `OpenCode step started.`.

## Actual behavior

WeSight shows that no local login/key is detected and the OpenCode task stalls immediately after starting.

This looks like a WeSight-side credential detection / OpenCode process integration issue rather than an OpenCode authentication issue.
